etl/tests/conftest.py

```diff
@@ -89,54 +103,40 @@ def sample_fixture() -> str:
 
 
 @pytest.fixture
-def azure_container() -> str:
+def azure_container_name() -> str:
     return 'test-container'
 
 
 @pytest.fixture
-def azure_service(azure_container: str) -> blob.BlockBlobService:
-    reader.EXTRACT_ENDPOINT_STORAGE_ACCOUNT = 'devstoreaccount1'
-    reader.EXTRACT_ENDPOINT_STORAGE_KEY = 'Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSR' \
-                                          + 'Z6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=='
-    reader.EXTRACT_ENDPOINT_STORAGE_DOMAIN = 'http://127.0.0.1:10000/devstoreaccount1'
-    reader.EXTRACT_ENDPOINT_CONTAINER = azure_container
-
-    azure_service = blob.BlockBlobService(account_name='devstoreaccount1',
-                                          account_key='Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSR'
-                                          + 'Z6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==',
-                                          custom_domain='http://127.0.0.1:10000/devstoreaccount1')
-    azure_service.create_container(azure_container)
+def azure_service(azure_container_name: str,
+                  monkeypatch: _pytest.monkeypatch.MonkeyPatch) -> typing.Iterator[blob.BlockBlobService]:
+    if not azure_emulator_is_running():
+        pytest.skip("Azure emulator is not running.")
```

Comment:
yeah, this is nice, having the skip happen automatically if the test asks for an Azure related fixture.